lighthouse-core/lib/page-functions.js

```diff
@@ -155,15 +155,33 @@ function getOuterHTMLSnippet(element, ignoreAttrs = [], snippetCharacterLimit =
     for (const attributeName of clone.getAttributeNames()) {
       if (charCount > snippetCharacterLimit) {
         clone.removeAttribute(attributeName);
-      } else {
-        let attributeValue = clone.getAttribute(attributeName);
-        if (attributeValue === null) continue;
-        if (attributeValue.length > ATTRIBUTE_CHAR_LIMIT) {
-          attributeValue = attributeValue.slice(0, ATTRIBUTE_CHAR_LIMIT - 1) + '…';
-          clone.setAttribute(attributeName, attributeValue);
+        continue;
+      }
+
+      let attributeValue = clone.getAttribute(attributeName);
+      if (attributeValue === null) continue; // Can't happen.
+
+      let dirty = false;
+
+      // Replace img.src with img.currentSrc.
+      if (attributeName === 'src' && element.tagName === 'IMG') {
```

Comment:
```suggestion
      if (attributeName === 'src' && 'currentSrc' in element) {
```

bonus.. also get this behavior for `<audio>` and `<video>` elems.